setup/scripts/run_docker_tests.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
ditto, here; Perhaps it's not a big deal, but the file naming here is not consistent.  Do you want to make it so as `docker_runner_tests.sh`?

Reply:
Reverted to your original suggestion: `run_docker_tests.sh` matching the naming convetion of the other scripts (verb_name.sh).